config/api.yml

```diff
@@ -2647,7 +2658,7 @@
         :identifier: picture_new
   :policies:
     :description: Policies
-    :identifier: miq_policy
+    :identifier: miq_policy,,
```

Comment:
I think we can support an Array here directly?

Reply:
will move these under `:vms: => :resource_collections: => :reconfigure:`